File: Yes.java
import java.io.BufferedOutputStream;
import java.io.IOException;

public class Yes {
    public static void main(String[] args) {
        final int MAX_BUFFER_SIZE = 65536;
        String inputString = args.length > 1 ? args[1] + '\n' : "y\n";

        StringBuilder bufferString = new StringBuilder();
        for (int i = MAX_BUFFER_SIZE; i > 0; i -= inputString.length()) {
            bufferString.append(inputString);
        }

        byte[] byteOutput = bufferString.toString().getBytes();
        try (BufferedOutputStream os = new BufferedOutputStream(System.out)) {
            while(true) 
                os.write(byteOutput);
        } catch (IOException e) {
            System.err.println(e.getLocalizedMessage());
        }
    }
}

File: Yes.cs
using System;
using System.IO;
using System.Text;

public class Yes {
    public static void Main(string[] args) {
        const int MAX_BUF_SIZE = 65536;
        int _buffer = MAX_BUF_SIZE;
        string _input = args.Length > 0 ? args[0] : "y";
        StringBuilder _stringBuffer = new StringBuilder(1024, MAX_BUF_SIZE);
        while (_buffer > 0) {
            _stringBuffer.Append(_input).Append('\n');
            _buffer -= (_input.Length + 1);
        }

        using (BufferedStream _stream = new BufferedStream(Console.OpenStandardOutput())) {
            byte[] _byteBuffer = Encoding.ASCII.GetBytes(_stringBuffer.ToString());
            try {
                while (true) _stream.Write(_byteBuffer, 0, MAX_BUF_SIZE);
            } catch (IOException e) {
                Environment.Exit(1);
            }
        }
    }
}
